configure.ac

```diff
@@ -697,9 +697,9 @@ if test "x$with_norm_ext" != "xno"; then
         if test "x$with_norm_ext" != "xyes"; then
             norm_path="${with_norm_ext}"
             norm_CFLAGS="${norm_CFLAGS} -I${norm_path}/include"
-            norm_LIBS="${norm_LIBS} -L${norm_path}/lib"
+            norm_LIBS="${norm_LIBS} -L${norm_path}/lib -L${norm_path}/protolib/lib"
         fi
-        norm_LIBS="${norm_LIBS} -lnorm"
+        norm_LIBS="${norm_LIBS} -lnorm -lprotokit"
```

Comment:
This is only for static linked builds though?

Reply:
That is what I have tested with, as it's the default NORM/protolib build method.  If there is an alternative method to supply that information via configure to allow compilation to complete, that would work as well, but I didn't see one.

Reply:
Ubuntu 18.04 only ships the shared library:

https://packages.ubuntu.com/bionic/amd64/libnorm-dev/filelist

So the build would fail there, as there's no libprotokit.a